data/styles/clear/include/Icons.mapcss

```diff
@@ -59,6 +59,9 @@ node|z12-[leisure=nature_reserve],
 area|z12-[leisure=nature_reserve]
 {text: name;text-color: @park_label;text-halo-opacity: 0.8;text-halo-color: @label_halo_light;text-halo-radius:0.5;}
 
+area|z10-[boundary=aboriginal_lands]
```

Comment:
On the other hand, if we're going to include them into the World map then it might make sense to display labels there - as there is usually more space (less visible features) and it could be confusing to see some pattern fill w/o a way to understand what is it..